pages/common/assets/sass/style.scss

```diff
@@ -512,3 +517,11 @@ details {
     vertical-align: top;
   }
 }
+
+#submitted-version {
+  .declarations {
+    .question span {
+      font-weight: bold;
```

Comment:
`<strong>` makes them bold without having to add extra styles

Reply:
We re-use the component in two places with different styling - in the activity log it's designed as non-bold and in the "Latest submission" section it's designed as bold, so this was the easiest way to get both.

Reply:
seems legit